themes/harmony/assets/reviews.js

```diff
@@ -105,7 +103,7 @@ const setupReviews = async () => {
    * Append reviews to the reviewsWrapper.
    */
   const addReviews = (reviewsWrapper, reviews) => {
-    reviewsWrapper.append(...reviews.map(review => createReviewItem(review)));
+    reviewsWrapper.append(...convertDate(reviews).map(review => createReviewItem(review)));
```

Comment:
Q: what is the reason for this change?

Reply:
hmm, I tried to revert to the previous code and it is functioning properly! 
PS: Aura didn't include this change.